roles/as_code/lib/lead_engineer.rb

```diff
@@ -2,7 +2,60 @@
 require_relative './shared_expectations/delivery_lead_expectations'
 
 JobSpec::Role.definition 'Lead Engineer' do
-  description 'Our Lead Engineers take responsibility for the successful delivery of software products and services. Experts in both software craftsmanship and agile delivery methods, Lead Engineers nurture engineering teams and are trusted partners to our customers.'
+  description <<~DESCRIPTION
+    Our Lead Engineers turn our customer’s needs into roadmaps, coach teams to deliver, are cutting the best code of their career, still learn everyday, and constantly strive to be nice humans.
```

Comment:
Everyday
Everyday (as one word) is an adjective. Thesauruses list average, mundane, ordinary, and standard as synonyms. “Everyday clothing,” then, refers to the ordinary clothes you wear on regular days, as opposed to outfits designated for special events or holidays. Occasionally, people use everyday as a noun—it’s a shorthand way of referring to their everyday routines.

Every Day
Every day means “each day.” The easiest way to remember this is to think about the space separating the two words. Because of that space, “every” is simply an adjective modifying the word “day.” If you paired every with any other word, it would mean each—every day means “each day”, just like “every word” means “each word.” There’s nothing more to it.